redbeams-api/src/main/java/com/sequenceiq/redbeams/api/endpoint/v4/databaseserver/requests/AllocateDatabaseServerV4Request.java

```diff
@@ -24,7 +24,9 @@
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class AllocateDatabaseServerV4Request extends ProviderParametersBase {
 
-    @Size(max = 40, min = 5, message = "The length of the name must be between 5 to 40 inclusive")
+    public static final int RDS_NAME_MAX_LENGTH = 40;
+
+    @Size(max = RDS_NAME_MAX_LENGTH, min = 5, message = "The length of the name must be between 5 to 40 inclusive")
```

Comment:
min?

Reply:
Sorry, could you please describe?

Reply:
constant for max length constraint was extracted, as opposed to min lenght constraint

Reply:
Thanks. Only the max length was reused in the code elsewhere.

Reply:
The message still has 40 hard coded.